src/ServiceControl.Monitoring/Infrastructure/EndpointTracker.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace ServiceControl.Monitoring.Infrastructure
+{
+    using System.Threading.Tasks;
+    using Metrics.Raw;
+    using NServiceBus;
+    using NServiceBus.Metrics;
+
+    public class EndpointTracker : IHandleMessages<MetricReport>, IHandleMessages<LongValueOccurrences>, IHandleMessages<Occurrences>
```

Comment:
As I mentioned above, I would rather trap `IMessage` and accidentally track something we didn't mean to, than forget to add a handler in here and miss tracking something in a weird edge case.